pkg/operator/ceph/disruption/clusterdisruption/osd.go

```diff
@@ -273,15 +273,21 @@ func (r *ReconcileClusterDisruption) updateNoout(pdbStateMap *corev1.ConfigMap,
 			}
 			if time.Since(nooutSetTime) >= r.maintenanceTimeout {
 				// noout expired
-				osdDump.UpdateFlagOnCrushUnit(r.context.ClusterdContext, false, namespace, failureDomain, nooutFlag)
+				if _, err := osdDump.UpdateFlagOnCrushUnit(r.context.ClusterdContext, false, namespace, failureDomain, nooutFlag); err != nil {
+					logger.Errorf("failed to update flag on crush unit when noout expired. %v", err)
```

Comment:
@rohantmp Should we return these errors instead of just logging them?

Reply:
Yes, that makes sense!